lighthouse-core/scripts/legacy-javascript/run.js

```diff
@@ -0,0 +1,313 @@
+  'es6.math.expm1',
+  'es6.math.fround',
+  'es6.math.hypot',
+  'es6.math.imul',
+  'es6.math.log10',
+  'es6.math.log1p',
+  'es6.math.log2',
+  'es6.math.sign',
+  'es6.math.sinh',
+  'es6.math.tanh',
+  'es6.math.trunc',
+  'es6.number.constructor',
+  'es6.number.epsilon',
+  'es6.number.is-integer',
+  'es6.number.is-safe-integer',
+  'es6.number.max-safe-integer',
+  'es6.number.min-safe-integer',
+  'es6.number.parse-float',
+  'es6.number.parse-int',
+  'es6.object.assign',
+  'es6.object.create',
+  'es6.object.define-properties',
+  'es6.object.define-property',
+  'es6.object.freeze',
+  'es6.object.get-own-property-descriptor',
+  'es6.object.get-own-property-names',
+  'es6.object.get-prototype-of',
+  'es6.object.is-extensible',
+  'es6.object.is-frozen',
+  'es6.object.is-sealed',
+  'es6.object.keys',
+  'es6.object.prevent-extensions',
+  'es6.object.seal',
+  'es6.object.set-prototype-of',
+  'es6.object.to-string',
+  'es6.promise',
+  'es6.reflect.apply',
+  'es6.reflect.construct',
+  'es6.reflect.define-property',
+  'es6.reflect.delete-property',
+  'es6.reflect.get-own-property-descriptor',
+  'es6.reflect.get-prototype-of',
+  'es6.reflect.get',
+  'es6.reflect.has',
+  'es6.reflect.is-extensible',
+  'es6.reflect.own-keys',
+  'es6.reflect.prevent-extensions',
+  'es6.reflect.set-prototype-of',
+  'es6.reflect.set',
+  'es6.set',
+  'es6.string.code-point-at',
+  'es6.string.ends-with',
+  'es6.string.from-code-point',
+  'es6.string.includes',
+  'es6.string.iterator',
+  'es6.string.raw',
+  'es6.string.repeat',
+  'es6.string.starts-with',
+  'es6.string.trim',
+  'es6.typed.array-buffer',
+  'es6.typed.data-view',
+  'es6.typed.float32-array',
+  'es6.typed.float64-array',
+  'es6.typed.int16-array',
+  'es6.typed.int32-array',
+  'es6.typed.int8-array',
+  'es6.typed.uint16-array',
+  'es6.typed.uint32-array',
+  'es6.typed.uint8-array',
+  'es6.typed.uint8-clamped-array',
+  'es6.weak-map',
+  'es6.weak-set',
+  'es7.array.includes',
+  'es7.object.entries',
+  'es7.object.get-own-property-descriptors',
+  'es7.object.values',
+  'es7.string.pad-end',
+  'es7.string.pad-start',
+];
+
+/**
+ * @param {{group: string, name: string, code: string, babelrc?: *}} options
+ */
+async function createVariant(options) {
+  const {group, name, code, babelrc} = options;
+  const dir = `${VARIANT_DIR}/${group}/${name.replace(/[^a-zA-Z0-9]+/g, '-')}`;
+
+  if (!fs.existsSync(`${dir}/main.bundle.js`) && (STAGE === 'build' || STAGE === 'all')) {
+    fs.mkdirSync(dir, {recursive: true});
+    fs.writeFileSync(`${dir}/main.js`, code);
+    fs.writeFileSync(`${dir}/.babelrc`, JSON.stringify(babelrc || {}, null, 2));
+    // Not used in this script, but useful for running Lighthouse manually.
+    // Just need to start a web server first.
+    fs.writeFileSync(`${dir}/index.html`, `<title>${name}</title><script src=main.bundle.min.js>`);
+
+    // Note: No babelrc will make babel a glorified `cp`.
+    execFileSync('yarn', [
```

Comment:
what's the expected cwd, `lighthouse-core/scripts/legacy-javascript`? let's assert that or set `cwd` appropriately since most scripts assume repo root is `cwd`